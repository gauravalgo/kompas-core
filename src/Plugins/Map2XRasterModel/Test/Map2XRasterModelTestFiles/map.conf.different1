version=3

model=Map2XRasterModel
tileSize=128 256

area=0 0 1 1
zoom=0
layer=base
