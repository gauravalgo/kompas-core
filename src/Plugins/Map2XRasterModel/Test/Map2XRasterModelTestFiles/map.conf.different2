version=3

model=Map2XRasterModel
tileSize=256 256
zoomStep=4

area=0 0 1 1
zoom=0
layer=base
