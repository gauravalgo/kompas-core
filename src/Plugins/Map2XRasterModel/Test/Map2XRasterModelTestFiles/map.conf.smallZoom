version=3

model=Map2XRasterModel
tileSize=256 256

area=18 6 1 1
zoom=3

layer=base
