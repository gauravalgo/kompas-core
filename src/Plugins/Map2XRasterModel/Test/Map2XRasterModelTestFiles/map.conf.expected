version=3
model=Map2XRasterModel
tileSize="256 256"
area="6 7 2 2"
zoom=2
zoom=3
layer=base
overlay=relief
name="Map2X testing package"
description="Buggy and sparse"
packager=mosra
